src/commands/PlayCommand.ts

```diff
@@ -109,80 +158,34 @@ export class PlayCommand extends BaseCommand {
                     this.client.logger.error("SP_PLAYLIST_ERR:", new Error(e.stack));
                     return message.channel.send(createEmbed("error", `I could not load the playlist\nError: **\`${e.message}\`**`));
                 }
-            } else {
+            } else if (urlData.type === "album") {
                 try {
                     const albumData = await getData(url);
-                    const playlistSearchStrings: string[] = await albumData.tracks.items.map((item: any): string => `${item.artists[0].name} - ${item.name}`);
+                    const playlistSearchStrings: string[] = await albumData.tracks.items.map((item: any): string => `${item.artists[0].name} - ${item.name} Audio`);
                     this.client.logger.debug(playlistSearchStrings.join("\n"));
                     const addingPlaylistVideoMessage = await message.channel.send(
-                        createEmbed("info", `Adding all music in **[${albumData.name}](${albumData.external_urls.spotify})** playlist, please wait...`)
+                        createEmbed("info", `Adding all music in **[${albumData.name}](${albumData.external_urls.spotify})** album please wait...`)
```

Comment:
According to line 51
```suggestion
                        createEmbed("info", `Adding all music in **[${albumData.name}](${albumData.external_urls.spotify})** album, please wait...`)
```